src/compensations/Compensations.tsx

```diff
@@ -2,30 +2,35 @@
 import styles from "./compensations.module.css";
 import Text from "src/components/text/Text";
 import { CompensationsPage } from "studio/lib/payloads/compensations";
-import { RichText } from "src/components/richText/RichText";
 import SalaryCalculator, {
   Degree,
 } from "./components/salaryCalculator/SalaryCalculator";
 import { useState } from "react";
 import { calculatePension, calculateSalary } from "./utils/calculateSalary";
+import { CompanyLocation } from "studio/lib/payloads/companyDetails";
+import {
+  IOption,
+  RadioButtonGroup,
+} from "src/components/forms/radioButtonGroup/RadioButtonGroup";
 
 interface CompensationsProps {
   compensations: CompensationsPage;
+  locations: CompanyLocation;
 }
 
 interface SalaryCalculatorFormState {
   examinationYear: number;
   selectedDegree: Degree;
 }
 
-const Compensations = ({ compensations }: CompensationsProps) => {
+const Compensations = ({ compensations, locations }: CompensationsProps) => {
+  const [selectedLocation, setSelectedLocation] = useState<string>("trondheim");
```

Comment:
```suggestion
  const [selectedLocation, setSelectedLocation] = useState<CompanyLocation>(locations[0]);
```

This isn't the best way, but let's just make sure Trondheim is the first in the company locations list in the studio and say that that's our default. For now.

Reply:
Agreed. We should avoid hardcoding based on specific Sanity data that might not be present (e.g. for Sweden). Picking the first one is more general, but in the future we probably want some "Default Location" field in Sanity.